README.md

```diff
@@ -1,2 +1,104 @@
-# ntrip-catalog
+# NTRIP-catalog
 Catalog of NTRIP providers with CRS information
+
+## What is NTRIP-catalog
+NTRIP-catalog is an open source and open data repository with the CRS information from multiple NTRIP service providers.
+The data is stored as JSON files, that can be easily parsed by any software.
+
+## What is the problem with coordinate reference systems in the NTRIP and RTCM protocols?
+Neither RTCM messages, nor the NTRIP handshake include any clarification about the cordinate reference system (CRS) that applies to the corrected coordinates.
```

Comment:
```suggestion
Neither RTCM messages (as of version 3.3 of the standard), nor the NTRIP handshake include any clarification about the cordinate reference system (CRS) that applies to the corrected coordinates.
```
What about adding the version? It makes the sentence unambiguously correct without making guesses about the version for which we don't have a copy of the standard.